projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -118,11 +119,13 @@ export class OAuthLibWrapperService {
    */
   initLoginFlow() {
     if (
-      !this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
+      !this.featureConfigService.isEnabled(
+        'nativeSupportForRedirectingOAuthFlows'
+      ) ||
+      this.authConfigService.getOAuthFlow() ===
+        OAuthFlow.ResourceOwnerPasswordFlow
```

Comment:
This change is related to the enhancements we made to support oAuth flows involving a redirect.

When the support is enhanced, we want to set the localstorage item on ROPC flow.  
When enhanced support is off, we want to always set the localstorage item.